ims/einstein/ceph.py

```diff
@@ -84,9 +84,21 @@ def list_images(self):
         return self.rbd.list(self.context)
 
     @log
-    def create_image(self, img_id, img_size):
+    def create_image(self, img_id, img_size, old_format=True, features=0):
+        """
+        Create a rbd image
+
+        :params img_id: what the image is called
+        :params img_size: how big the image is in bytes
+        :params old_format: whether to create an old-style image that is
+                            accessible by old clients, but cannot user more
+                            advanced features like layering
+        :params features: bitmask of features to enable
+        :return Boolean
+        """
         try:
-            self.rbd.create(self.context, img_id, img_size)
+            self.rbd.create(self.context, img_id, img_size,
+                            old_format=old_format, features=features)
```

Comment:
Based on my test, we are able to create an image supporting layering when we set old_format = False and features=1